app/models/booking.rb

```diff
@@ -18,8 +19,6 @@ class Booking < ActiveRecord::Base
 
   belongs_to :tx, class_name: "Transaction", foreign_key: "transaction_id"
 
-  attr_accessible :transaction_id, :end_on, :start_on
```

Comment:
This is removed because `attr_accessible` is deprecated in favor of `strong_params`